content/tutorials/guides/how-to-collect-product-variations.md

```diff
@@ -297,58 +428,102 @@ In the guided tutorial <a href="/tutorials/how-to-get-families-and-attributes.ht
 
 ```php [activate:PHP]
 
-function getProductModels(): array
-{
-    $client = buildApiClient();
-
-    $maxProductsPerPage = 100;
-    $maxFamiliesPerQuery = 3;
-    $scope = 'ecommerce';
-
-    // Get family codes from storage
-    $familyCodes = getFamilyCodes();
-    // Get locales from storage
-    $locales = getLocales(); // ['en_US', 'fr_FR']
-
-    $familyCodeChunks = array_chunk($familyCodes, $maxFamiliesPerQuery);
-
-    $apiUrl = '/api/rest/v1/product-models?'
-        . 'locales=%s'
-        . '&scope=%s'
-        . '&search={"family":[{"operator":"IN","value":%s}],"parent":[{"operator":"EMPTY"}]}'
-        . '&limit=' . $maxProductsPerPage;
-
-
-    // Collect product models from API
-    $productModels = [];
-    foreach ($familyCodeChunks as $familyCodes) {
-        $response = $client->get(
-            sprintf(
-                $apiUrl,
-                implode(',', $locales),
-                $scope,
-                json_encode($familyCodes)
-            )
-        );
-        $data = json_decode($response->getBody()->getContents(), true);
-        $productModels[] = $data['_embedded']['items'];
-    }
+$client = buildApiClient();
+
+$maxProductsPerPage = 100;
+$maxFamiliesPerQuery = 10;
+$scope = 'ecommerce';
```

Comment:
Get from the storage (in PHP snippet also)

Reply:
we decided to fix this into refacto ticket